package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "react-date-time-range",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "directories": {
+    "doc": "doc",
+    "test": "test"
+  },
+  "scripts": {
+    "ci": "npm run lint && npm run coverage",
+    "coverage": "istanbul cover _mocha -- --compilers js:babel/register --require react-tests-globals-setup test/test* && istanbul check-coverage --statements 95 --branches 95 --functions 95 --lines 95",
+    "prepublish": "mkdir -p dist && babel src/DateTimeRange.jsx > dist/DateTimeRange.js",
+    "prestart": "ulimit -n 9999",
+    "start": "watchify -t babelify -t brfs doc/example.js -o doc/example-built -v",
+    "test": "mocha --require react-tests-globals-setup --compilers js:babel/register test/test*",
+    "lint": "make-up src test doc"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "brfs": "^1.4.1",
+    "insert-css": "^0.2.0",
+    "react": "^0.13.0",
+    "react-bootstrap": "^0.26.1",
+    "react-date-time-group": "shackpank/react-date-time-group",
```

Comment:
It's expecting `main` to be some JS generated pre-publish to npm, only the jsx is committed into the repository. This will generate the js:
```
cd node_modules/react-date-time-group
npm run prepublish
```